### PR TITLE
Add devices permissions

### PR DESCRIPTION
**Pull request by poupotte**
DON'T MERGE PLEASE

### CODE REVIEW

File: server/controllers/applications.coffee [hunk 1]
```diff
@@ -113,13 +115,16 @@ updateApp = (app, callback) ->
                     console.log err
                     iconInfos = null
                 data.iconType = iconInfos?.extension or null
-                app.updateAttributes data, (err) ->
+                # Update access
+                app.updateAccess access, (err) ->
                     callback err if err?
```
Comment (aenario) [inline server/controllers/applications.coffee]: should be return callback

File: test/applications_tests.coffee [hunk 2]
```diff
@@ -139,9 +172,37 @@ describe "Applications management", ->
...
+                ds = new Client "http://localhost:9101/"
+                ds.setBasicAuth 'my-app', @password
+                data =
+                    docType: 'contact'
+                    slug: "blabla"
+                ds.post 'data/', data, (err, res, body) ->
+                    should.not.exist body.error
+                    done()
+
+            it 'Then application can access to contact', (done) ->
```
Comment (aenario) [inline test/applications_tests.coffee]: change test's description

File: test/applications_tests.coffee [hunk 3]
```diff
@@ -103,6 +106,36 @@ describe "Applications management", ->
...
```
Comment (aenario) [inline test/applications_tests.coffee]: change tests description

File: test/applications_tests.coffee [hunk 4]
```diff
@@ -198,36 +259,36 @@ describe "Applications management", ->
...
-                , 3.1 * 60 * 1000
-
-            it "And controller have been requested to stop this app", ->
-                request = @controller.lastCall().request
-                request.url.should.equal "/drones/my-app/stop"
-                request.method.should.equal "POST"
 
-            it "And the proxy have been requested to update its routes", ->
-                @proxy.lastCall().request.url.should.equal "/routes/reset"  ###
+            it 'Then application can access to contact', (done) ->
```
Comment (aenario) [inline test/applications_tests.coffee]: change tests description.

Why is this code three time in the test suite ?

File: test/autostop.coffee [hunk 5]
```diff
@@ -441,48 +441,58 @@ describe "Auto-stop manager", ->
...
-                    Event: description: 'access the events'
                 isStoppable: true
+            @client.post "api/applications/install", app, (err, res, app) =>
+                #console.log err
+                #console.log app
+                #@client.put "api/application/byId/#{app.id}", app, (err) ->
+                done()
 
-            app.save done
+        it "Then I wait for 4 minutes", (done) ->
```
Comment (aenario) [inline test/autostop.coffee]: Why the extratimer ? and change its description.

File: test/applications_tests.coffee (hunk 4 above)
Comment (poupotte) [inline test/applications_tests.coffee]: We test application permissions after : 
* installation
* update
* start/stop
Application document or access document can be changed after an update or restart (as an example : password are changed after a restart).

**Pull request closed without merging**